queue/sqs.go

```diff
@@ -0,0 +1,220 @@
+package queue
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/aws/aws-sdk-go/service/sqs/sqsiface"
+	"github.com/google/uuid"
+	"log"
+	"strconv"
+)
+
+const (
+	EnqueueBatchLimit = 10 // Max number of messages SQS will let you enqueue at once
```

Comment:
nit: SQSBatchLimit bc the queue package is a generic queue interface and this limit is inherit to SQS queue?
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-batch-api-actions.html
Batch operations all seem to have the same limit, I couldn't find the receive limit though

Reply:
Agree, updated